### DISCUSSION TITLE
4.2.2 Board Added

### DISCUSSION THREAD

**Discussion by LeeOtts** [LeeOtts/Ender3v2-Klipper-Configs#17] [Announcements]
A few people have opened discussions asking about a 4.2.2 board. 

I do not have a 4.2.2 board, so I have not been able to test this. Based on what other users have said, changing the pins should work, so that is what I have done. 

To use it, change the name to printer.cfg. It should work with the macros.cfg and the KAMP files.

**Comment by edanielc** [comment]
Thank you very much, I'm starting with 3D printer and... I really apreciate this. Just right now is printing my first temp tower. 
![WhatsApp Image 2024-09-02 at 22 22 11_facf346a](https://github.com/user-attachments/assets/b76d48f4-0fca-408a-88d6-4fece664370e)
